### PR TITLE
reduce inrepoconfig cache size defaults

### PR DESCRIPTION
**Pull request by listx**
It has been observed in practice that the gerrit component, under high load (where many of the changes being processed are using inrepoconfig), can run out of memory before reaching the 1000 keys limit. In our internal instance we were hitting ~470 keys before getting OOMKilled.

/cc @airbornepony @timwangmusic @cjwagner

### CODE REVIEW

**Comment by listx** [comment]
Before we merge this let's also first test that using 400 in our internal instance is actually OK (we'll be testing cache key evictions for the first time, I think).

**Comment by listx** [comment]
Update: we're now testing with 200 as the default. Seems to be OK. Updated to use 200 in this PR also.

File: prow/flagutil/config/config.go
Comment (cjwagner) [inline prow/flagutil/config/config.go:60]: "Each host receives its own cache."  Is this still true?

**Review by cjwagner — APPROVED** [review]
/hold

File: prow/flagutil/config/config.go
Comment (listx) [reply prow/flagutil/config/config.go]: Done.

**Review by cjwagner — APPROVED** [review]
/unhold

**Pull request merged**
